include/stumpless/option.h

```diff
@@ -71,6 +71,15 @@
 #    define STUMPLESS_OPTION_NOWAIT (1<<4)
 #  endif
 
+/** Log the message to the stderr as well as
+ *  to the target
+*/
+# ifdef STUMPLESS_SYSLOG_H_COMPATIBLE
+#   define STUMPLESS_OPTION_PERROR LOG_PERROR
```

Comment:
I did not realize this had not been defined yet, good catch! Please add an entry for `STUMPLESS_OPTION_PERROR` in `tools/check_headers/stumpless.yml` so that it will be picked up by the header checking tool.